crossdock/setup_crossdock.py

```diff
@@ -20,6 +20,7 @@
         ]
     },
     install_requires=[
-        # all dependencies are included in tchannel already
+        # most of dependencies are included in tchannel already
+        'six'
```

Comment:
Got it, will fix. 
I guess I just don't entirely understand what this crossdock thing is all about :)